### ISSUE TITLE
Empty folder `concat-stats-for`

### ISSUE THREAD

**Issue by xg-wang** [broccolijs/broccoli-concat#136]
Repro:

```
ember new test-app
cd test-app
npm i
ember b
```

@simonihmig

**Comment by simonihmig** [comment]
Just to make sure, before I dig into this: the last line should have been `CONCAT_STATS=true ember b`, right?

**Comment by xg-wang** [comment]
@simonihmig No, setting the env flag will leave debug info in the folder. I'm on ember-cli 3.3.0

**Comment by simonihmig** [comment]
@xg-wang Not sure I understand the problem, can you please elaborate? 

Without setting `CONCAT_STATS=true` nothing is written to `./concat-stats-for`, by intention.

> setting the env flag will leave debug info in the folder.

This sound as if it is actually working for you, is it?

**Comment by xg-wang** [comment]
Before `ember b`
```
 ❯ ls                                                                                                                                                                  [08:06:02]
README.md          config             node_modules       package.json       testem.js          vendor
app                ember-cli-build.js package-lock.json  public             tests
```

After
```
❯ ls                                                                                                                                                                  [08:06:29]
README.md          concat-stats-for   dist               node_modules       package.json       testem.js          tmp
app                config             ember-cli-build.js package-lock.json  public             tests              vendor
```

There's an empty `concat-stas-for`.

**Comment by simonihmig** [comment]
Ah, ok, got it! 😂 Will investigate...

I assume this is using the latest release (3.7.0)? (`npm ls broccoli-concat`)

**Comment by simonihmig** [comment]
Fixed in #137
